src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;

```

Comment:
Let's check common mistakes file 

Your project structure should consist the following packages:

db for holding Storage
model for holding models like Fruit (if necessary)
service for holding services, like Writer, Reader, Parser and so on
service.impl for holding implementations of services
strategy for holding handlers for different operations (you are expected to apply Strategy pattern)

So replace `Storage` to db package